src/nuget/xunit.console.netcore.nuspec

```diff
@@ -33,11 +33,21 @@
       <dependency id="System.Threading" version="4.0.10" />
       <dependency id="System.Threading.Tasks" version="4.0.10" />
       <dependency id="System.Xml.XDocument" version="4.0.10" />
+      <dependency id="xunit.runner.utility" version="2.1.0-rc1-build3168" />
     </dependencies>
   </metadata>
   <files>
     <!-- xunit runner for .NET Core -->
     <file src="xunit.console.netcore\xunit.console.netcore.exe" target="lib\aspnetcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dnxcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dotnet" />
+    
+    <!--
+      NuGet ignores .exe files in lib folders, so use .props files to copy the exe
+      to the output folder.
+    -->
+    <file src="xunit.console.netcore\xunit.console.netcore.exe" target="build\dotnet" />
+    <file src="xunit.console.netcore\xunit.console.netcore.props" target="build\dotnet" />
   </files>
```

Comment:
After some investigation, it looks like the bug is with the NuGet restore command, not the ResolveNuGetPackageAssets task.  I've filed a bug (NuGet/Home#1454) and sent a PR (NuGet/NuGet3#132) to fix it.

In the meantime we need a solution to deploy the xUnit .NET Core Console runner for MSBuild.  Can we go ahead and merge this PR to unblock us?  I think this is the cleanest solution until the NuGet bug is fixed.  NuGet v3 has improved support for .props and .targets files and no longer needs to modify your project files to include them.